src/order/max.lean

```diff
@@ -66,33 +100,98 @@ several top elements. When `α` is linear, this is useful to make a case disjunc
 `no_max_order α` within a proof. -/
 def is_top (a : α) : Prop := ∀ b, b ≤ a
 
-lemma is_top.to_dual (h : is_top a) : is_bot (to_dual a) := h
-lemma is_bot.to_dual (h : is_bot a) : is_top (to_dual a) := h
+/-- `a` is a minimal element of `α` if no element is strictly less than it. We spell it without `<`
+to avoid having to convert between `≤` and `<`. -/
```

Comment:
```suggestion
to avoid having to convert between `≤` and `<`: this conversion can be found in `is_min_iff_forall_not_lt`. -/
```
and similarly for max